React-frontend/src/components/Admin/TaskCard.js

```diff
@@ -93,17 +98,53 @@ function TaskCard({ id, title, description, due_on, is_completed }) {
                 }
             </Typography>
 
-            {/* Button to mask task completed */}
             {
                 (auth.user.role === "admin") ?
-                    (<Button
-                        variant="contained"
-                        color="primary"
-                        onClick={() => toggleComplete(id, is_completed)}>
+                    (<div>
+                        {/* Task Completed status Button */}
                         {
-                            (is_completed) ? (<span>Mark Incomplete</span>) : (<span>Mark Complete</span>)
+                            (is_completed) ? (
+                                <Tooltip title="Mark as Not Completed" placement="bottom-start">
+                                    <Button
+                                        variant="contained"
+                                        color="primary"
+                                        onClick={() => toggleComplete(id, is_completed)}
+                                        className={classes.button}
+                                    >
+                                        <CheckCircleIcon
+                                            className={classes.icons}
+                                        />
+                                    </Button>
+                                </Tooltip>
+                            ) : (
+                                <Tooltip title="Mark as Completed" placement="bottom-start">
+                                    <Button
+                                        variant="contained"
+                                        color="primary"
+                                        onClick={() => toggleComplete(id, is_completed)}
+                                        className={classes.button}
+                                    >
+                                        <CheckCircleOutlineIcon
+                                            className={classes.icons}
+                                        />
+                                    </Button>
+                                </Tooltip>
+                            )
                         }
-                    </Button>)
+                        {/* Task Edit Button */}
+                        <Tooltip title="Edit" placement="bottom-end">
+
+                            <Button
+                                variant="contained"
+                                color="primary"
+                                onClick={() => { console.log('edit task', id) }}
```

Comment:
Is it necessary to console log on Click?

Reply:
That onClick was added for Debugging purposes and was left there for when the TaskEdit (#288) feature has been added. But for now I have removed it and also resolved an unnecessary warning from the console.